app/controllers/public.js

```diff
@@ -20,6 +20,19 @@ export default class PublicController extends Controller {
     return this.session.currentRouteName && this.session.currentRouteName !== 'public.cfs.new-session' && this.session.currentRouteName !== 'public.cfs.new-speaker' && this.session.currentRouteName !== 'public.cfs.edit-speaker' && this.session.currentRouteName !== 'public.cfs.edit-session';
   }
 
+  @computed('locationName', 'online')
```

Comment:
Wrong computed properties

Every event is shown as Online Event. Please test the PR properly

Reply:
@iamareebjamal It shows online event at first, but when you refresh it shows the correct value. Then this value shows up for the next event you click. Can you help me, I am not able to figure it out myself.

Reply:
`@computed('model.locationName', 'model.online')`

Reply:
@iamareebjamal  Thanks